lesson01/LESSON1.md

```diff
@@ -1,5 +1,41 @@
-### Setting up your tools
+### Checking your Computer
+Before any of the necessary software is downloaded, we need to make sure you have an appropriate computer that supports 
+our version of Minecraft and Forge. A good rule of thumb is start off with a computer that can run ordinary Minecraft pretty well.
 
-### IntelliJ
+Forge runs on Windows(7 and greater), and on Mac(10.8.3 and greater). However, all operating systems must be 64-bit. 
+Chromebooks are not supported.
 
-### Minecraft Plugin
\ No newline at end of file
+### Setting up your tools 
+#### Java
+We will be using Kotlin as our programming language, but Kotlin still generates Java bytecode when compiled. This means 
+we will need an appropriate Java JDK. The version of Java we need is the latest version of [1.8](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html).
+Scroll down on that link and select either Windows x64 or macOS x64 depending on your operating system. You may need to
+create an account to install the JDK.
+
+#### IntelliJ
+IntelliJ is the IDE we will be using for programming. It handles a lot of the heavy lifting for plugins and dependencies we will need for
```

Comment:
I would rewrite this to "IntelliJ is an IDE(Integrated Development Environment) that we will use to write Kotlin code and create our mods." Don't worry about mentioning dependencies, lets introduce that term later in the camp once the kids have gotten more comfortable with the rest of the jargon they'll be learning.